upup/pkg/fi/cloudup/dotasks/loadbalancer.go

```diff
@@ -163,6 +163,8 @@ func (_ *LoadBalancer) RenderDO(t *do.DOAPITarget, a, e, changes *LoadBalancer)
 		return err
 	}
 
+	klog.Infof("Load balancer created with ID=%s", loadbalancer.ID)
```

Comment:
```suggestion
	klog.V(2).Infof("Load balancer created with ID=%s", loadbalancer.ID)
```

Reply:
@olemarkus - Strangely, I don't see the logs emitted if I use `klog.V(2)` when testing locally.
However, I do see if with `klog.Infof` as seen below

`I1114 10:36:38.306548    5486 loadbalancer.go:166] Load balancer created with ID=59a14963-08b5-4cfb-94cd-2ffd00d9b6e1`

Am I missing something here?